src/components/DayPickerKeyboardShortcuts.jsx

```diff
@@ -110,6 +157,7 @@ class DayPickerKeyboardShortcuts extends React.Component {
     return (
       <div>
         <button
+          id="showKeyboardShortcutsButton"
```

Comment:
it seems like this ID only exists so it can be targeted in tests; if so, please remove it - there's surely another way it can be located.

Reply:
Ok I'll remove and update the test(s)